packages/babel-plugin-transform-property-literals/src/index.js

```diff
@@ -12,6 +12,11 @@ module.exports = function({ types: t }) {
             return;
           }
 
+          const containsUnicode = key.value.match(/[^a-z0-9$_]/i);
```

Comment:
Since you don’t care about the actual matches, let’s use `test` instead of `match`.